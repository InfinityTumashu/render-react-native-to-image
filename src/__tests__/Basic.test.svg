<?xml version="1.0" encoding="UTF-8" ?>
<svg width="320" height="480" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1">

 <rect type="View" x="0.5" y="0.5" width="319" height="479" fill="none" stroke="red" stroke-width="1"/>

 <g transform='translate(0, 0)'>
   <image xlink:href="./example.png" preserveAspectRatio="" x="136" y="76" width="48" height="48"/>
   <text><tspan x="127" y="136.7802734375" fill="black" font-family="'Helvetica'" font-weight="normal" font-style="normal" font-size="14">Hello folks</tspan></text>
   <rect type="View" x="135" y="142" width="50" height="50" fill="powderblue"/>
   <text><tspan x="112" y="204.7802734375" fill="black" font-family="'Helvetica'" font-weight="normal" font-style="normal" font-size="14">Ok </tspan>
<tspan fill="black" font-family="'Helvetica'" font-weight="bold" font-style="normal" font-size="14">here</tspan>
<tspan fill="black" font-family="'Helvetica'" font-weight="normal" font-style="normal" font-size="14"> is</tspan>
<tspan x="112" y="222.7802734375" fill="black" font-family="'Helvetica'" font-weight="normal" font-style="normal" font-size="14">something </tspan>
<tspan fill="black" font-family="'Helvetica'" font-weight="normal" font-style="italic" font-size="14">long</tspan>
<tspan x="112" y="240.7802734375" fill="black" font-family="'Helvetica'" font-weight="normal" font-style="italic" font-size="14">that </tspan>
<tspan fill="black" font-family="'Helvetica'" font-weight="normal" font-style="normal" font-size="14">is going to</tspan>
<tspan x="112" y="258.7802734375" fill="black" font-family="'Helvetica'" font-weight="normal" font-style="normal" font-size="14">wrap for me.</tspan></text>
   <rect type="View" x="135" y="264" width="50" height="50" fill="skyblue"/>
   <rect type="View" x="135" y="314" width="50" height="50" fill="steelblue"/>
   <rect type="View" x="140" y="364" width="40" height="40" fill="green"/>

   <g transform='translate(140, 364)'>
     <rect type="View" x="10" y="10" width="20" height="20" fill="blue"/>
   </g>

 </g>

</svg>
