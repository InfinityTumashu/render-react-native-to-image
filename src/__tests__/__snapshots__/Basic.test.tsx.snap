// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counting nodes it is good with memory 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "borderColor": "red",
      "borderWidth": 1,
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "./example.png",
      }
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Hello folks
  </Text>
  <View
    style={
      Object {
        "backgroundColor": "powderblue",
        "height": 50,
        "width": 50,
      }
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "maxWidth": 100,
      }
    }
  >
    Ok 
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    >
      here
    </Text>
     is something 
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "fontStyle": "italic",
        }
      }
    >
      long that 
    </Text>
    is going to wrap for me.
  </Text>
  <View
    style={
      Object {
        "backgroundColor": "skyblue",
        "height": 50,
        "width": 50,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "steelblue",
        "height": 50,
        "width": 50,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "green",
        "padding": 10,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "blue",
          "padding": 10,
        }
      }
    />
  </View>
</View>
`;
